### PR TITLE
[MTA-2227] Review for the "Introduction to the MTA" guide

### PR DESCRIPTION
**Pull request by mpershina**
Extensive review for the [Introduction to the Migration Toolkit for Application](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.2/html/introduction_to_the_migration_toolkit_for_applications/index) guide.

- [Jira tracker](https://issues.redhat.com/browse/MTA-2227)
- [Gdoc](https://docs.google.com/document/d/1udIfXSFIZV3Ek6FutspMalyhB9kwiNM_CvUSuM5URRk/edit) for comments and suggestions (**Note:** Suggestions made in this PR can differ from those in the gdoc)

### CODE REVIEW

File: docs/getting-started-guide/master.adoc [hunk 1]
```diff
@@ -9,29 +9,29 @@ include::topics/templates/document-attributes.adoc[]
...
 
-== Supported configurations
+//== Supported configurations
 
 // Supported Migration Paths
-include::topics/migration-paths.adoc[leveloffset=+2]
+include::topics/migration-paths.adoc[leveloffset=+1]
 
 // Prerequisites
 == Prerequisites
```
Comment (mpershina) [inline docs/getting-started-guide/master.adoc:37]: This module is somewhat confusing. Can we at least say what are these prerequisites for? MTA installation? Needs clarification.

File: docs/topics/migration-paths.adoc [hunk 2]
```diff
@@ -4,15 +4,13 @@
 
 :_content-type: CONCEPT
 [id="migration-paths_{context}"]
-= Supported migration paths
+= Supported {ProductName} migration paths
 
-The {ProductName} ({ProductShortName}) supports migrations from third-party enterprise application servers, such as Oracle WebLogic Server, to JBoss Enterprise Application Platform (JBoss EAP) and upgrades to the latest release of JBoss EAP.
+The {ProductName} ({ProductShortName}) supports migrations from third-party enterprise application servers, such as Oracle WebLogic Server, to JBoss Enterprise Application Platform (JBoss EAP), and upgrades to the latest release of JBoss EAP.
```
Comment (mportman12) [inline docs/topics/migration-paths.adoc:9]: It's really hard to follow this list. Recommend reformatting as bullet points.

Comment (mpershina) [reply docs/topics/migration-paths.adoc:9]: Nice! Will incorporate in the next commit

File: docs/getting-started-guide/master.adoc (hunk 1 above)
Comment (anarnold97) [reply docs/getting-started-guide/master.adoc:37]: Grand so, my only concern here is the errors in Pantheon, which happened last time a writer tried to change the name

If we can agree to change the name for 7.1 when i build this release in Pantheon then we should be fine

But changing it now might cause some problems.

If you are really determined to change it now I will not stop you... but let us not change before Tuesday as I need to publish Release Notes for 7.0.2

Comment (mpershina) [reply docs/getting-started-guide/master.adoc:37]: Ok, got it, Andy! Let's then wait until Tuesday when you're done with Release Notes.

Comment (anarnold97) [reply docs/getting-started-guide/master.adoc:37]: Cool... grand with me, you can make an absolute mess of the repo so it never works again but only AFTER Tuesday. Just be prepared to take the blame and me to say "Radhika! Masha done broke Pantheon and ran away"  :rofl:
